aws-traffic-mirroring/README.md

```diff
@@ -0,0 +1,105 @@
+Setup for Seekret Deployment with Amazon VPC Traffic Mirroring.
+
+**Table of Contents**
+1. [Introduction](#introduction)
+2. [Prerequisites](#prerequisites)
+3. [Deployment](#deployment)
+   1. [Sample Service](#sample-service)
+   2. [Target Instance](#target-instance)
+   3. [VPC Mirroring](#vpc-mirroring)
+4. [Analyzing and Capturing Mirrored Traffic](#analyzing-and-capturing-mirrored-traffic)
+5. [Additional Features](#additional-features)
+   1. [HTTPS Listener for Sample Service
+](#https-listener-for-sample-service)
+   2. [Mirror Filters for Application Load Balancers
+](#mirror-filters-for-application-load-balancers)
+6. [Cleanup](#cleanup)
+7. [Sources, References & Additional Material](#sources-references-&-additional-material)
+
+## Introduction
+
+This repository contains a setup for Seekret's sniffer deployed along Amazon VPC Traffic Mirroring.
+
+Amazon VPC Traffic Mirroring allows you to mirror network traffic from an Elastic Network Interface (ENI) to another ENI (or a Network Load Balancer). See [AWS Documentation](https://docs.aws.amazon.com/vpc/latest/mirroring/what-is-traffic-mirroring.html) for additional information.
+
+**Note**: Amazon VPC Traffic Mirroring supports ENIs of instances running on top of the AWS Nitro System (e.g. t3, m5, c5 and r5 instances). You cannot mirror traffic from an ENI that is attached to a non-Nitro instance.
+
+The repository contains the following components (or CloudFormation stacks):
+
+* `vpc-mirroring-target-instance` - EC2 instance with Seekret's sniffer.
+* `vpc-mirroring` - Amazon VPC Traffic Mirroring configuration.
+
+## Prerequisites
+
+The setup in this repository has the following requirements:
+
+* AWS CLI with admin-level credentials (needs to be able to deploy IAM roles).
+
+Also, the `Makefile` is optimized for deployments made on `us-east-1` region (but should work with other regions as well).
+
+## Deployment
+
+### Target Instance
+
+Deploy the target instance stack by running
+
+You'll need to provide parameter values for the next parameters
+
+```
+CustomerVpcId - VPC Id where your ALB resides
+SourceVpcIpv4Cidr - VPC IPv4 CIDR
+CustomerSubnetId - Subnet of the availability zone of the ALB
+```
+
+```bash
+aws --profile default --region us-east-1 cloudformation deploy --stack-name seekret-sniffer \ 
```

Comment:
does the profile is always default?

Reply:
good question, i think we can leave it like this for now

Reply:
does guesty have a default profile?

Reply:
i don't know, but if not I'll tell them to change it before deployment. would you prefer a mention in the readme?

Reply:
yes, both for profile and region

Reply:
fixed